modules/microshift-cri-o-container-runtime.adoc

```diff
@@ -26,9 +39,32 @@ Environment=HTTPS_PROXY="http://$PROXY_USER:$PROXY_PASSWORD@$PROXY_SERVER:$PROXY
 $ sudo systemctl daemon-reload
 ----
 
-. Restart the CRI-O service to apply the settings:
+. Restart the CRI-O service:
 +
 [source,terminal]
 ----
 $ sudo systemctl restart crio
 ----
+
+. Restart the {microshift-short} service to apply the settings:
++
+[source,terminal]
+----
+$ sudo systemctl restart microshift
+----
+
+.Verification
+
+. Verify that pods are started by running following command and examining the output:
++
+[source,terminal]
+----
+$ oc get all -A
+----
+
+. Verify that {microshift-short} is able to pull container images by running the following command and examining the output:
++
+[source,terminal]
+----
+$ crictl images
```

Comment:
This looks like it is a command that needs to be executed by root (or using `sudo`). If not, there is an error

`[redhat@dhcp-1-235-98 ~]$  crictl images
FATA[0000] unable to determine image API version: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial unix /var/run/crio/crio.sock: connect: permission denied"`